api/v1beta1/policyautomation_types.go

```diff
@@ -94,3 +114,24 @@ type ClusterEvent struct {
 func init() {
 	SchemeBuilder.Register(&PolicyAutomation{}, &PolicyAutomationList{})
 }
+
+// ReplicatedDetailsPerTemplate defines the replicated policy compliance details and history
+type ReplicatedDetailsPerTemplate struct {
+	// +kubebuilder:pruning:PreserveUnknownFields
+	ComplianceState policyv1.ComplianceState      `json:"compliant,omitempty"`
+	History         []ReplicatedComplianceHistory `json:"history,omitempty"`
+}
+
+// ReplicatedComplianceHistory defines the replicated policy compliance details history
+type ReplicatedComplianceHistory struct {
+	LastTimestamp metav1.Time `json:"lastTimestamp,omitempty" protobuf:"bytes,7,opt,name=lastTimestamp"`
+	Message       string      `json:"message,omitempty" protobuf:"bytes,4,opt,name=message"`
+}
+
+// ReplicatedPolicyStatus defines the replicated policy status
+type ReplicatedPolicyStatus struct {
+	// +kubebuilder:validation:Enum=Compliant;NonCompliant
```

Comment:
Could you please remove this kube builder marker since this struct isn't generated into a CRD?